src/utils/setAuthToken.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const setAuthToken = (token) => {
+  if (token) {
+    axios.defaults.headers.common['x-auth-token'] = token;
```

Comment:
headers is fine to store the token but you can also store it in the user's localstorage. What ever is more secure, I don't mind.